src/js/components/Chat/Chat.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from "react";
+import '../../../scss/components/Chat/Chat.scss';
+
+class Chat extends Component {
+  render() {
+    return (
+      <main className="chat">
+        <div className="chat-top">
+          <nav className="nav-top">
+            <div className="nav-group nav-group-top">
+              <div className="nav-item"/>
+              <div className="nav-item nav-item-grow"/>
+              <div className="nav-item"/>
+              <div className="nav-item"/>
+              <div className="nav-item"/>
+              <div className="nav-item"/>
+            </div>
+          </nav>
+        </div>
+        <div className="chat-body"/>
+        <div className="chat-foot"/>
+      </main>
+    );
+  }
+}
+
+export default Chat;
```

Comment:
Did this component need to be part of the PR?

Reply:
Not really, it was one of those "distractions". I need to stop doing that